src/iconc/cmain.c

```diff
@@ -121,10 +123,23 @@ char **argv;
    fprintf(stderr,"iconc library files found in %s\n",refpath);
 #else					/* ExpTools */
    if ((int)strlen(patchpath) > 18)
-      refpath = patchpath+18;
+      refpath = patchpath + 18;
 #endif					/* ExpTools */
    else
-     refpath = relfile(argv[0], "/../");
+      refpath = relfile(argv[0], "/../");
+
+   /*
+      if (refpath[strlen(refpath)] != '/') {
+       unsigned int n = (unsigned int)strlen(refpath) + 2;
+       char * refpathtmp = (char *)alloc(n);
+       strncpy(refpathtmp, refpath, n-1);
+       refpath = refpathtmp;
+       strcat(refpath, "/");
+     }
+    */
+
+   // find the path to the top level directory
+   refpath = unirootfile(argv[0], "/../../");
```

Comment:
@cjeffery, when you browse the changes om github, you can click the small "plus" sign button that appears next to a line when  you hover over it to a add comment tat references that line. You can always select then drag to comment on a whole block like what I'm doing here. 

The  last line I added `refpath = unirootfile(argv[0], "/../../");` replaces all lines from 104 to 140. I don't know what ExpTools is, or if I can safely drop all of that code.